### PR TITLE
Feat/key management service b0

### PR DESCRIPTION
**Pull request by vovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13837
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

This the technical base for Key Management Service.
It does not add any real features but add the uapp in the manager with feature flipping.
It has been requested to be pushed to prod to ensure that everything works with feature availability and applications on 2API.

### CODE REVIEW

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/key-management-service/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-key-management-service-app' --include-dependencies -- npm run dev:watch --if-present",
+    "test:e2e": "tsc && npx playwright test --headed",
+    "test:e2e:ci": "tsc && npx playwright test",
+    "test:e2e:script": "tsc && node ../../../../scripts/run-playwright.js"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^7.3.0",
+    "@ovh-ux/manager-core-api": "*",
+    "@ovh-ux/manager-core-utils": "*",
+    "@ovh-ux/manager-react-core-application": "*",
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/package.json:27]: Why do you need this package ?

File: packages/manager/apps/key-management-service/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "@ovh-ux/manager-react-shell-client": "^0.4.0",
+    "@ovh-ux/manager-tailwind-config": "*",
+    "@ovh-ux/shell": "^3.1.0",
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/package.json:30]: I don't think you use this package directly

File: packages/manager/apps/key-management-service/package.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "@ovhcloud/manager-components": "*",
+    "@ovhcloud/ods-common-core": "17.1.0",
+    "@ovhcloud/ods-common-theming": "^17.1.0",
+    "@ovhcloud/ods-components": "^17.1.0",
+    "@ovhcloud/ods-theme-blue-jeans": "^17.1.0",
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/package.json:35]:
```suggestion
    "@ovhcloud/ods-common-core": "17.2.1",
    "@ovhcloud/ods-common-theming": "17.2.1",
    "@ovhcloud/ods-components": "17.2.1",
    "@ovhcloud/ods-theme-blue-jeans": "17.2.1",
```

File: packages/manager/apps/key-management-service/src/api/GET/apiv2/services.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
+import { queryClient } from '@ovh-ux/manager-react-core-application';
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/src/api/GET/apiv2/services.ts:1]: Don't import queryClient directly please

File: packages/manager/apps/key-management-service/src/shell.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  });
+
+  return shellClient;
+};
+
+export const getShellClient = () => {
+  return shellClient;
+};
+
+export default { setShellClient, getShellClient };
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/src/shell.ts:18]: this file is unused I think

File: packages/manager/apps/key-management-service/src/index.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  );
+
+  context.shell.i18n.onLocaleChange(({ locale }: { locale: string }) => {
+    if (reloadOnLocaleChange) {
+      window.top?.location.reload();
+    } else {
+      i18n.changeLanguage(locale);
+    }
+  });
+
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/src/index.tsx:39]: You can use the initI18n from react-shell-client

File: packages/manager/apps/key-management-service/src/App.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { createHashRouter, RouterProvider } from 'react-router-dom';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import { odsSetup } from '@ovhcloud/ods-common-core';
+import { useShell } from '@ovh-ux/manager-react-shell-client';
+import '@ovhcloud/ods-theme-blue-jeans';
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/src/App.tsx:7]: You could move this import in the index where there is the import of another global.css

File: packages/manager/apps/key-management-service/src/api/GET/apiv2/services.ts [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+  fetchIcebergV6,
+  apiClient,
+} from '@ovh-ux/manager-core-api';
+
+type PermissionsGroup = unknown;
+type Response = unknown;
+type Uuid = unknown;
+type Action = unknown;
+type Resource = unknown;
+type TagFilter = unknown;
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/src/api/GET/apiv2/services.ts:13]: test

**Review by darsene — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**